src/integrationTest/java/com/mongodb/hibernate/BasicInsertionTests.java

```diff
@@ -115,12 +116,8 @@ void testEntityWithEmbeddedFieldInsertion(SessionFactoryScope scope) {
     }
 
     private void onMongoCollection(Consumer<MongoCollection<BsonDocument>> collectionConsumer) {
-        var connectionString = new ConnectionString(new Configuration().getProperty(JAKARTA_JDBC_URL));
-        try (var mongoClient = MongoClients.create(MongoClientSettings.builder()
-                .applyConnectionString(connectionString)
-                .build())) {
-            var collection =
-                    mongoClient.getDatabase(connectionString.getDatabase()).getCollection("books", BsonDocument.class);
+        try (var mongoClient = MongoClients.create(config.mongoClientSettings())) {
```

Comment:
We now use `MongoConfiguration` instead of reading and parsing `JAKARTA_JDBC_URL` directly.